test/test-frames/EMU_EMU-Professional-375-M-Bus.xml

```diff
@@ -241,7 +241,7 @@
     <DataRecord id="30">
         <Function>Instantaneous value</Function>
         <StorageNumber>0</StorageNumber>
-        <Unit>Unrecognized VIF extension: 0x60</Unit>
+        <Unit>Reset counter</Unit>
```

Comment:
I prefer to have updates to the test frames as a separate commit.

Reply:
I am used to have automatic tests being executed, so I did update the code and the tests at the same time.
Do you insist on splitting it into separate commits?

Reply:
Does it mean #165 enforces this?

I didn't come a final decision about this, but i'm concerned about possible corner cases.

Reply:
Yes, #165 would enforce that you have to update the tests when you change the code.

Can you describe a corner case?

I'm quite new to all this, there could be some magic string that you can put in the commit message in order to skip running the CI jobs.

My concern is that I don't want to try to do a change, and find out that the tests are not in sync with the code. ("It was broken when I got it")

Reply:
I'm not against automatic tests. I'm only concerned that the tests apply on commit level and not on pull request level.

Unfortunately i don't have a specific corner case in mind. On a negative side this could lead to the situation that contributors tend to put everything in a single commit.

So at least for this pull request, i'm fine with it.

Reply:
I'm not sure how github works. I think that github run tests on the latest commit (on a branch or pull request).

Once you are actually uesd to it, it seems natural, when I do see a code update, I expect to find corresponding updates in the tests. It is very seldom that we do change the xml-format an will have to update every test-file.